resources/diameter-s13/du/pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.mobicents.servers.diameter.resources</groupId>
+		<artifactId>s13-parent</artifactId>
+		<version>1.0.0.FINAL</version>
```

Comment:
Please use <version>2.8.0-SNAPSHOT</version>
and add <relativePath>../..</relativePath>